tests/test_serving.py

```diff
@@ -171,3 +172,21 @@ def real_app(environ, start_response):
     r = requests.get(server.url)
     assert r.status_code == 200
     assert r.content == b'hello'
+
+
+def test_monkeypached_sleep(tmpdir):
+    script = tmpdir.mkdir('app').join('test.py')
+    script.write(textwrap.dedent('''
+    import time
+
+    def sleep(secs):
+        pass
+
+    # simulate eventlet.monkey_patch by replacing the builtin sleep
+    # with a regular function
+    time.sleep = sleep
+
+    from werkzeug._reloader import ReloaderLoop
+    ReloaderLoop()._sleep(0)
+    '''))
+    subprocess.check_call(['python', str(script)])
```

Comment:
I don't think this test is useful, as all it does is sleep when the method didn't get patched properly.

Also, you can safely do monkeypatching with [pytest](http://pytest.org/latest/monkeypatch.html), no need to spawn a new process.